monitor-iot-data-with-freeboard/README.md

```diff
@@ -0,0 +1,204 @@
+
+- Get **freeboard** from github:
+
+```console
+$ git clone https://github.com/Freeboard/freeboard.git
+```
+
+- Get Kuzzle Feeboard Datasource plugin
+
+```console
+$ git clone https://github.com/etrousset/kuzzle-freeboard-plugin.git
+```
+
+It is recommanded to clone both repositories from the same folder so that you don't have to adapt paths from when following the steps.
+You should end up with a folder containing the following folders:
+
+```console
+.
+├── freeboard
+└── kuzzle-freeboard-plugin
+```
+
+## Howto steps
+
+### Step 0: Getting Kuzzle Reading
+
+Assuming you already have an intance of Kuzzle running on *localhost*, we will need to add some fixtures. You will need to create a new *index* and *collection*.
+
+In `scripts` folder, you can launch `fb-kuzzle-fixtures.sh` to create the required data structure. This will create an `fb-howto-index` index, and `fb-howto-collection` as shown bellow.
+
+![alt text](img/kuzzle-admin-console.png "Kuzzle freeboard fixture")
+
+### Step 1: Enable Kuzzle Datasource Plugin
+
+First we have to make Kuzzle JS sdk available to *freeboard* plugins.
+One way to do this, is to *kuzzle-sdk* it to *freeboard* depndencies using *npm*.
+
+In `freeboard` folder enter the following command:
+
+```console
+$ npm install kuzzle-sdk
+```
+
+Then modify `freeboard/index-dev.html` file to add loading of Kuzzle JS SDK, and Kuzzle DataSource plugin.
+
+``` html
+    <script type="text/javascript">
+        head.js("lib/js/thirdparty/knockout.js",
+                "lib/js/thirdparty/jquery.js",
+                "plugins/freeboard/freeboard.widgets.js",
+                .
+                .
+                .
+                "examples/plugin_example.js",
+
+                // Make Kuzzle() available
+                "node_modules/kuzzle-sdk/dist/kuzzle.js",
+                // Load Kuzzle Datasource plugin
+                "../kuzzle-freeboard-plugin/kuzzle_datasource.js",
+
+                // *** Load more plugins here ***
+                function(){
+                    $(function()
+                    .
+                    .
+```
+
+You can now check that Kuzzle DataSource plugin is available. Open `freeboard/index-dev.html` in your favorite web browser.
+Under *DATASOURCES*, click the *ADD* button. In the dropdown list, you should be able to see **Kuzzle Datasource** entry.
+
+![alt text](img/datasource-plugins.png "Kuzzle Datasource plugin")
+
+### Step 2: Kuzzle Datasource Configuration
+
+Once you clicked on **Kuzzle Datasource** entry, you get to the plugin settings screen.
+
+![alt text](img/kuzzle-datasource-settings.png "Kuzzle Datasource settings")
+
+Settings:
+
+| Name | Description |
+|----------------------|------|
+|Kuzzle host  | The hostname of the machine on which Kuzzle Server instance si running, default is *localhost*. |
```

Comment:
`is running`

Reply:
I think that `localhost` should be put between backticks instead of been italicized. That remark applies to all other default values below.

Reply:
Nitpicking?
Seeing how easy it is to configure this how-to for any Kuzzle Server instance, not just one running locally, I would drop all "running locally" mentions in this how-to